tests/common/test_downstream_fatal_error.py

```diff
@@ -0,0 +1,57 @@
+"""Test downstream fatal error."""
+
+import unittest
+from unittest.mock import Mock, patch
+
+import __init__
+
+import tests.common.test_common_constants as TestCommonConstants
+from pyracf import DownstreamFatalError, UserAdmin
+from pyracf.common.irrsmo00 import IRRSMO00
+
+# Resolves F401
+__init__
+
+
+@patch("pyracf.common.irrsmo00.IRRSMO00.call_racf")
+class TestDownstreamFatalError(unittest.TestCase):
+    maxDiff = None
+    IRRSMO00.__init__ = Mock(return_value=None)
+    user_admin = UserAdmin()
+
+    def test_security_request_error_thrown_on_precheck_error(
```

Comment:
`downstream_fatal_error` not `security_request_error`

Reply:
fixed in all locations here.